compiler/Makefile

```diff
@@ -176,6 +176,12 @@ test-frontend: FORCE $(CHPL_CONFIG_CHECK) $(COMPILER_BUILD)
 	  cd $(COMPILER_BUILD)/frontend/test && ctest $$JOBSFLAG . ;
 	@echo "frontend tests are available in build/frontend-test"
 
+frontend-tests: FORCE $(CHPL_CONFIG_CHECK) $(COMPILER_BUILD)
+	@echo "Making the frontend tests..."
+	@cd $(COMPILER_BUILD) && $(CMAKE) $(CHPL_MAKE_HOME) $(CMAKE_FLAGS_NO_NDEBUG) && $(MAKE) tests
+	@echo "Making symbolic link to frontend tests in build/frontend-test"
+	@cd ../build && rm -f frontend-test && ln -s $(COMPILER_BUILD)/frontend/test frontend-test
+
```

Comment:
There's some code duplication here with `test-frontend` (no surprises there). More significantly, I don't think there's a reason to mark this target as `FORCE`. It already depends on `COMPILER_BUILD`, which I will assume includes the Dyno object files. The reason that `test-frontend` is marked `FORCE` is, I assume, to actually _run_ the tests every time you call `make`, even if nothing changed. I think you could:

1. Change `test-frontend` to list `frontend-tests` as a dependency.
2. Remove the code from `test-frontend` to build and link the frontend tests
3. Remove `FORCE` from `frontend-tests` to avoid redundant work.

Of these, the third might be questionable because of the symbolic link part (do we want to re-link every time?), but the first two seem to have no downsides to me.

Reply:
done!